.github/styles/Vocab/Base/accept.txt

```diff
@@ -128,4 +128,5 @@ write_mode
 skip_tables
 subfolder
 recurse
-CCPA's
\ No newline at end of file
+CCPA's
+sqlite
```

Comment:
```suggestion
SQLite
```
These are case sensitive, so I think we want `SQLite` as that's the name of the product